internal/pkg/print/print.go

```diff
@@ -112,3 +127,21 @@ func (p *Printer) PromptForConfirmation(prompt string) error {
 	}
 	return fmt.Errorf("max number of wrong inputs")
 }
+
+// Shows the content in the command's stdout using the "less" command
+// If output format is set to none, it does nothing
+func (p *Printer) PagerDisplay(content string) error {
```

Comment:
Could we have a simple test for this function also? Similar to the ones for the `Output...` functions?